doc/guides/maintaining-npm.md

```diff
@@ -0,0 +1,50 @@
+# Maintaining npm in Node.js
+
+## Step 1: Clone npm
+
+    $ git clone https://github.com/npm/npm.git
```

Comment:
Nit: can we use the triple ` instead of the indentation? I think it's more common in our guides.

Reply:
<pre>
```console
$ git clone https://github.com/npm/npm.git
```
</pre>

Would be nice